app/views/layouts/design_system.html.erb

```diff
@@ -0,0 +1,38 @@
+<% environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment %>
+
+<%= render "govuk_publishing_components/components/layout_for_admin", product_title: "Publisher", browser_title: yield(:page_title), environment: environment  do %>
```

Comment:
This line is quite long, maybe we should consider putting the arguments on separate lines for readability.